File: Src/Client/src/app/shared/models/user.model.ts
import { AppRolesEnum } from './user-app.model';
export class UserModel {
    id: number;
    email: string;
    firstName: string;
    lastName: string;
    positionId: number;
    sex: EnumUserSex;
    doB: Date;
    phoneNo: any;
    roleId: AppRolesEnum;
    status: boolean;

    constructor(init?: Partial<UserModel>) {
        Object.assign(this, init);
    }
}

export enum EnumUserSex {
    MALE = 1,
    FEMALE = 2
}

File: Src/Server/Kloon.EmployeePerformance.Models/User/UserModel.cs
using Kloon.EmployeePerformance.Models.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kloon.EmployeePerformance.Models.User
{
    public class UserModel
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int PositionId { get; set; }
        public SexEnum Sex { get; set; }
        public DateTime? DoB { get; set; }
        public string PhoneNo { get; set; }
        public int RoleId { get; set; }
        public bool Status { get; set; }
        public string FullNameEmail
        { 
            get 
            {
                return FirstName + " " + LastName +" (" + Email + ")"; 
            }
        }
        public string FullName
        {
            get
            {
                return FirstName + " " + LastName;
            }
        }
    }
}
